website/templates/project/registration_editor_templates.mako

```diff
@@ -72,6 +72,7 @@
               <div class="form-group" data-bind="css: {has-success: $data.isComplete}">
                 <span data-bind="with: $root.context($data)">
                   <span data-bind="if: $root.showValidation">
+                    <p data-bind="validationMessage: $data.value, style: { color: '#ff0033' }"></p>
```

Comment:
Use bootstrap whenever possible rather than hand-coded colors, styles, etc.
http://getbootstrap.com/2.3.2/base-css.html#typography
See the ```text-*``` classes